CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 - Prevent deletion of snapshots that are backing searchable snapshot indexes ([#5069](https://github.com/opensearch-project/OpenSearch/pull/5069))
 - Add max_shard_size parameter for shrink API ([#5229](https://github.com/opensearch-project/OpenSearch/pull/5229))
+- Support request level durability for remote store enabled indices([#5699](https://github.com/opensearch-project/OpenSearch/pull/5699))
```

Comment:
Is there a top-level issue we can link to that better describes the "request level durability" issue? Also please put a space before the link to match the style elsewhere in this file.

Reply:
Yes, we have #5671.